### ISSUE TITLE
Stripe inventory and pricing Gatsby plugins

### ISSUE THREAD

**Issue by kennedyrose** [smarterlabs/labs#43]

**Comment by tbaustin** [comment]
@kennedyrose would this sdk work? https://stripe.com/docs/api/skus/list

Can make it like the esca price/inventory stuff if you want? 

Also do you have test keys or should I just keep making my own for all these things

**Comment by kennedyrose** [comment]
Yeah, with React hooks. I don't think I have keys set up for this yet.

**Comment by tbaustin** [comment]
okay i'll just set up a test key with some test products!

**Comment by tbaustin** [comment]
@kennedyrose  this should be working to produce nodes for SSR. Since we have to use an sdk and pass a key the only other way i could think of using custom hooks would be to temporarily store the data in cache maybe?

https://github.com/tbaustin/labs/tree/master/packages/%40smarterlabs/gatsby-plugin-stripe-priceline
